po/es.po

```diff
@@ -135,19 +134,19 @@ msgstr "`break` y `continue`"
 
 #: src/SUMMARY.md src/control-flow-basics/blocks-and-scopes.md
 msgid "Blocks and Scopes"
-msgstr ""
+msgstr "Funciones"
 
 #: src/SUMMARY.md src/control-flow-basics/functions.md
 msgid "Functions"
 msgstr "Funciones"
 
 #: src/SUMMARY.md src/control-flow-basics/macros.md
 msgid "Macros"
-msgstr ""
+msgstr "Atajos (_Macros_)"
```

Comment:
I'm not sure about this. Translating "Macro" to "Atajo" sounds like we are talking about keyboard shortcuts, but we're not talking about this. Macro comes from Greek to mean large. Spanish also has this sort of meaning for macro and maybe there's no need to translate it. Feel free to suggest otherwise.